packages/services/src/workers/monitor-keeper.js

```diff
@@ -1,88 +1,90 @@
-import postmark from 'postmark'
+import { ServerClient } from 'postmark'
+import Etherscan from '../models/Etherscan'
 import Models from '@aragonone/court-backend-server/build/models'
 import Network from '@aragonone/court-backend-server/build/web3/Network'
 
-// notifications settings
-const FROM = 'noreply@aragon.one'
+import { fromWei } from 'web3-utils'
+import { bigExp } from '@aragonone/court-backend-shared/helpers/numbers'
+import getWalletFromPk from '@aragonone/court-backend-shared/helpers/get-wallet-from-pk'
 
-// number of blocks checked backwards first time
-const BLOCKS_BACKWARDS_INITIAL_THRESHOLD = 10
+const FROM = 'noreply@aragon.one'
+const BALANCE_THRESHOLD = bigExp(1, 17) // 0.1 ETH
 
-let lastCheckedBlockNumber = 0
+const { Admin, KeeperSuspiciousTransaction } = Models
 
 export default async function (logger) {
-  const court = await Network.getCourt()
-  const { environment } = court
-  const keeper = await getKeeperAddress(environment)
-  const courtAddresses = await getWhitelistedAddresses(court)
+  const { environment: { network } } = Network
+  const etherscan = new Etherscan(network, process.env.ETHERSCAN_API_KEY)
+  const keeper = getWalletFromPk(process.env.PRIVATE_KEY).getAddressString()
```

Comment:
Sanity checking—what happens here if we don't pass in a private key (e.g. during local development)?

Reply:
It fails :)